src/main/resources/test.properties

```diff
@@ -0,0 +1,3 @@
+url = https://trello.com/login
+key = 9db03ef220396c5981df9171424f8474
```

Comment:
Credentials should not be stored in the repository, please remove them

Reply:
Fixed this

Reply:
But key and token values are still in this file